### ISSUE TITLE
Please Help.

### ISSUE THREAD

**Issue by Coherent5845** [prashanthca/syncer#1]
Hey can you please tell me how can i add my mega account detail to the scrip so that i can also upload files to mega.

**Comment by prashanthca** [comment]
I see you've already added the login credentials in the mega.json file of your fork. This is the file which is used by the application to access MEGA. One small piece of request: Purge mega.json, .megacmd.json, account.js and megamail.py from your fork. I had accidentally checked these files in and they contain some sensitive information.

To do this, use the following commands:

`cd syncer`
`git rm --cached mega.json`
`git rm --cached .megacmd.json`
`git rm --cached megamail.py`
`git rm --cached account.js`
`git commit --amend -CHEAD`
`git push -f`

Once the push is complete, you can re-create mega.json with the following contents:
`
{
  "accounts": [
    {
      "username": "username",
      "password": "password",
      "name": "Account 1"
    }
  ]
}
`
And then, run this command:
`echo "mega.json" >> .gitignore`

This is very important as this will retain the mega.json in your working directory, but will not check it into github.

**Comment by Coherent5845** [comment]
@prashanthca will it work for heroku cause i have deployed it on heroku.

**Comment by prashanthca** [comment]
Yes. It will work on heroku. By the way, did you remove the files I've mentioned in my previous comment from your fork?

**Comment by Coherent5845** [comment]
Can you please tell me how can i deploy it to arukas/docker.
And please if possible then please make video of "How to add mega credits to the code" for deploying to heroku, i had tried but failed.
